ci/helpers.js

```diff
@@ -0,0 +1,88 @@
+const https = require('https')
+const { ECSClient, ListTasksCommand } = require('@aws-sdk/client-ecs')
+
+
+/**
+ * Returns list of running ECS Cloudwatch logs for running test tasks
+ * @param {Array<string>} taskArns 
+ * @returns {Array<string>}
+ */
+function generateDiscordCloudwatchLogUrls(taskArns) {
+  const arnRegex = /\w+$/
+
+  const LogUrls = taskArns.map((arn, index) => {
```

Comment:
Nitpick but nice to keep case the same as the rest of the code unless there is a reason to do otherwise.
```suggestion
  const logUrls = taskArns.map((arn, index) => {
```

Reply:
missed this one, updated.